### DISCUSSION TITLE
Is it compatible with 25q64fvaig?

### DISCUSSION THREAD

**Discussion by Junior1930** [nofeletru/UsbAsp-flash#82] [Q&A] [answered]
Good evening,

I can only find 25q64fv and not 25q64fvaig

**Comment by nofeletru** [comment]
Last letters for marking package types mostly.

**Comment by mitko-kamburow** [comment, marked as answer]
Yes, it is supported. Just select W25Q64FV and it will work. Please, mark this or nofeletru's answer to close the discussion.

**Reply by Junior1930** [reply]
Thank you
